src/components/Card.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { shallow } from 'enzyme';
+import Card from './Card';
+
+describe('Card', () => {
+  test('it will match the last snapshot', () => {
+    const card = shallow(
+      <Card id={0} text="" emoji="" deleteCardCallback={()=>{}}/>
+    );
+
+    expect(card).toMatchSnapshot();
+
+  });
+
+  test('it invokes a callback when the card is deleted', () => {
+    const callback = jest.fn();
+    const card = shallow(
+      <Card id={0} text="" emoji="" deleteCardCallback={callback}/>
+    );
+
+    card.find('button').simulate('click');
```

Comment:
This test is failing because you have a confirmation dialog as well.  You need to [research how to mock that](https://stackoverflow.com/questions/48728167/simulate-clicking-ok-or-cancel-in-a-confirmation-window-using-enzyme).